### ISSUE TITLE
GW2Launcher Blish Hud launch problem

### ISSUE THREAD

**Issue by Krasus1313** [Healix/Gw2Launcher#254]
Hi, when you suggested I could use "-m MumbleLink_%accountid%" to start Blish Hud in client start.
I did it in 1 account but actually It doesnt work properly when it is started by GW2 Launcher, It makes mouse movement quite weird and with multiple account launched it sometimes crash one of 2 accounts.

Top of it, I have to exit and restart it again.
Right now it starts with only 1 account but when I check account's options>gw2>launch options>runs after launching part, there isn't anything to delete to make it stop.

**Comment by Healix** [comment]
Run after launcher can be configured per account (Edit > Launch options) or for all accounts (Settings > Guild Wars 2 > Launch options). If you wanted it to start on all accounts, you add it to the settings, whereas if you only want it on specific accounts, you add it to each individual account.

Note that to use "MumbleLink_%accountid%" you'll also need to enable changing the mumble link name, which can be done per account or for all accounts under the launch options.

You should change Blish to run after loading a character, since it will only be wasting resources up until that point (because GW2 doesn't initialize mumble link until a character is loaded).

If by weird mouse movement you mean it's stuttered/skipping (as if it's low FPS) that will happen if your system is at 100% CPU usage and/or you've enabled the option to prioritize GW2 while initializing the game.

What is crashing? GW2? Depends what the crash says. One of the DX11 patches last year introduced a bug that can cause GW2 to freeze and/or crash while loading character select, which is more likely to occur if something else takes priority in that moment. You can only try to prevent this by prioritizing GW2 while initializing (Settings > Guild Wars 2) and delaying launches until the game is loaded (Settings > General > Launching).

**Comment by Krasus1313** [comment]
Alright from instructions I found how to change it. Now it doesn't launch it.

However how am I gonna enable to change of "MumbleLink_%accountid%" ?
And if "mumblelink" part is correct this is how it supposed to be? If not please let me know I'll change it.
![image_2023-02-05_185341215](https://user-images.githubusercontent.com/118668145/216829868-1a7bd9b5-5522-41a2-b772-e7dd4ae4f647.png)

And for mouse movement what I was trying to Say: Normally when you move camera with RMB, It disappears instantly and you can move mouse. 
But when this happens and when I try to move camera around while holding RMB, mouse still stays visible and move horizontally together with my camera movement and depending which side I'm moving.

Also by crash, I mean 1x Blish Hud program tries to activate itself on 2 accounts by switching its visibility in two at same time however it makes it glitchy and eventually causes crash time to time.

**Comment by Healix** [comment]
That is correct, assuming you only want it on one account (can't tell if Blish Hud was added to the account or settings, but that's the mumble link name under the account). Check "Run on the current Windows user" to prevent permission problems, if you happen to be using different users. Also ensure you haven't set -mumble in the command line arguments under either Edit > Launch options or Settings > Guild Wars 2 > Launch options.

If it's not starting on that account, launch the account, load a character, then right click the account > Selected > Run... > Blish Hud. It'll be displayed in blue if it's running. Clicking Blish Hud will start it, or it'll give you a reason why it can't be started (assuming you're using the latest version of Gw2Launcher; use beta to be sure, it's listed as build 7079 under Settings > Updates).

The mouse button staying visible would be a problem with an overlay. Overlays intercept your input and pass them on to GW2, so when something goes wrong, you'll get behavior like this, or your input may not work at all. I'm guessing that was caused by having multiple overlays on the same window, but it can also be caused by the overlay being too slow (100% CPU usage), or it could just be a bug with it.

Blish switching windows would be due to multiple clients using the same mumble link name. That shouldn't be a problem with the current configuration.
